src/style.css

```diff
@@ -0,0 +1,82 @@
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  width: 100%;
+  display: flex;
+  height: 70px;
+  margin-top: 20px;
+}
+
+.form-small {
+  height: 42px;
+}
+
+.label {
+  display: block;
+  position: relative;
+  width: 100%;
+}
+
+.label::before {
+  content: "";
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+  background-image: url("./images/Search.svg");
+  background-size: contain;
+  z-index: 1;
```

Comment:
Please, check and follow this approach [css-order](https://9elements.com/css-rule-order/), fix it in all places.